### PR TITLE
add new board config for fmu-v3

### PR DESCRIPTION
**Pull request by kd0aij**
This fmu-v3 config is the same as v2, except for allowing use of the entire 2MB flash memory, and including more modules in the build.

### CODE REVIEW

**Comment by dagar** [comment]
https://github.com/PX4/Firmware/issues/4910

**Comment by LorenzMeier** [comment]
@kd0aij Can you please not duplicate files when not needed? E.g. the src/drivers folder should not be required.

**Comment by kd0aij** [comment]
@LorenzMeier Sorry, this shouldn't have been a PR
@dagar Thanks, I was unaware of #4910

**Comment by LorenzMeier** [comment]
Well, I'd like to get this config in, but with a minimum duplication.

**Comment by kd0aij** [comment]
I'll submit a PR once I have something that will actually run on Pixhawk1/2. I'm assuming that I should be able to test on Pixhawk1, but I need to refresh my memory of the symptoms of the silicon errata.

File: cmake/configs/nuttx_px4fmu-v3_default.cmake [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+	# Board support modules
+	#
+	drivers/device
+	drivers/stm32
+	drivers/stm32/adc
+	drivers/stm32/tone_alarm
+	drivers/led
+	drivers/px4fmu
+	drivers/px4io
+	drivers/boards/px4fmu-v3
```
Comment (davids5) [inline cmake/configs/nuttx_px4fmu-v3_default.cmake]: drivers/boards/px4fmu-v2

**Pull request closed without merging**